docs/resources/bcs_instance.md

```diff
@@ -0,0 +1,352 @@
+  cluster_id            = data.huaweicloud_cce_cluster.test.id
+  consensus             = "kafka"
+  edition               = 4
+  eip_enable            = true
+  fabric_version        = "1.4"
+  enterprise_project_id = var.enterprise_project_id
+  password              = var.instance_password
+  volume_type           = "efs"
+  org_disk_size         = 500
+  database_type         = "couchdb"
+  orderer_node_num      = 2
+  bandwidth_size        = 5
+  delete_storage        = true
+  delete_obs            = true
+
+  couchdb {
+    user_name = var.database_user_name
+    password  = var.database_password
+  }
+  peer_orgs {
+    org_name = "organization01"
+    count    = 2
+  }
+  peer_orgs {
+    org_name = "organization02"
+    count    = 2
+  }
+  channels {
+    name      = "channel01"
+    org_names = [
+      "organization02",
+    ]
+  }
+  channels {
+    name      = "channel02"
+    org_names = [
+      "organization01",
+      "organization02",
+    ]
+  }
+  sfs_turbo {
+    share_type        = "STANDARD"
+    type              = "efs-ha"
+    flavor            = "sfs.turbo.standard"
+    availability_zone = data.huaweicloud_availability_zones.test.names[0]
+  }
+  kafka {
+    flavor            = "c3.mini"
+    storage_size      = 600
+    availability_zone = [
+      data.huaweicloud_availability_zones.test.names[0],
+      data.huaweicloud_availability_zones.test.names[1],
+      data.huaweicloud_availability_zones.test.names[2],
+    ]
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the instance.
+  If omitted, the provider-level region will be used.
+  Changing this will create a new instance.
+
+* `name` - (Required, String, ForceNew) Specifies a unique name of the BCS instance.
+  The name consists of 4 to 24 characters, including letters, digits, chinese charactors and hyphens (-), and the name
+  cannot start with a hyphen.
+  Changing this will create a new instance.
+
+* `edition` - (Required, Int, ForceNew) Specifies Service edition of the BCS instance.
+  Valid values are `1`, `2` and `4`.
+  For the specifications corresponding to these three versions, please refer to the [specification table](#jump).
+  Changing this will create a new instance.
+
+* `fabric_version` - (Required, String, ForceNew) Specifies version of fabric for the BCS instance.
+  Valid values are `1.4` and `2.0`
+  Changing this will create a new instance.
+
+* `consensus` - (Required, String, ForceNew) Specifies the consensus algorithm used by the BCS instance.
+  The valid values of fabric 1.4 are `solo`, `kafka` and `SFLIC`, and the valid values of fabric 2.0 are `SFLIC`
+  and `etcdraft`.
+  Changing this will create a new instance.
+
+* `orderer_node_num` - (Required, Int, ForceNew) Specifies the number of peers in the orderer organizaion.
+  For the number of orderer nodes, please refer to the [specification table](#jump).
```

Comment:
Is the specification table shown the **max** number of orderer nodes?

Reply:
it's both the maximum value and the initial number of creations at the time of creation.

Reply:
It's the minimum value of orderer nodes, the document has been updated.